Makefile

```diff
@@ -240,6 +240,9 @@ kube-dist: kube uaa-kube-dist
 	${GIT_ROOT}/make/kube-dist
 	rm -rf kube
 
+bundle-dist: kube-dist cert-generator 
```

Comment:
This means we always build the bundles here instead of in CI… actually that's probably a good idea anyway.